webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/ElementsTreeTools/ElementsTreeFilter.tsx

```diff
@@ -6,20 +6,12 @@
  * you may not use this file except in compliance with the License.
  */
 import { observer } from 'mobx-react-lite';
-import styled, { css } from 'reshadow';
 
-import { Filter, useFocus, useStyles, useTranslate } from '@cloudbeaver/core-blocks';
+import { Filter, s, useFocus, useS, useTranslate } from '@cloudbeaver/core-blocks';
 import type { ComponentStyle } from '@cloudbeaver/core-theming';
 
 import type { IElementsTree } from '../useElementsTree';
-
-const filterStyle = css`
-  filter-box {
-    composes: theme-background-surface from global;
-    padding: 8px 12px;
-    flex: 0 0 auto;
-  }
-`;
+import styles from './ElementsTreeFilter.m.css';
 
 interface Props {
   tree: IElementsTree;
```

Comment:
please remove `style?: ComponentStyle;` property from components if they don't use `useStyles` don't pass `style` to `useS` they is incompatible